x/claim/client/cli/tx.go

```diff
@@ -0,0 +1,38 @@
+package cli
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	// "github.com/cosmos/cosmos-sdk/client/flags"
```

Comment:
remove it?

Reply:
[695aa26](https://github.com/arkeonetwork/arkeo/pull/29/commits/695aa2671d3af12adf78bd728b14aafa1398889e)